modules/installation-guide/pages/using-the-openshift-web-console-to-configure-the-checluster-custom-resource-during-installation.adoc

```diff
@@ -0,0 +1,8 @@
+[id="using-the-openshift-web-console-to-configure-the-checluster-custom-resource-during-installation"]
+// = Using the OpenShift web console to configure the `CheCluster` Custom Resource during installation
+:navtitle: Using the OpenShift web console to configure the `CheCluster` Custom Resource during installation
```

Comment:
I can't find it, but I believe we've got a guideline to avoid fixed-width formatting in titles. If so, then maybe:
```suggestion
:navtitle: Using the OpenShift web console to configure the CheCluster Custom Resource during installation
```